sycl/test/invoke_simd/invoke_simd.cpp

```diff
@@ -35,9 +35,9 @@ ESIMD_CALLEE(float *A, esimd::simd<float, VL> b, int i) SYCL_ESIMD_FUNCTION {
   return a + b;
 }
 
-SYCL_EXTERNAL
-simd<float, VL> __regcall SIMD_CALLEE(float *A, simd<float, VL> b,
-                                      int i) SYCL_ESIMD_FUNCTION;
+[[intel::device_indirectly_callable]] SYCL_EXTERNAL
```

Comment:
Why is this extra attribute needed in addition to SYCL_EXTERNAL?

Reply:
FE requires this - functions whose addresses are taken need to have this attribute (seems reasonable?)

Reply:
It seems odd to require this from the user because they aren't taking a function pointer but our implementation is. You wrote in the other thread:

> Since this is still experimental, I think we can start with requiring the extra option, and then after some maturing fix the FE (which is pretty lengthy process)

This makes sense. At the same time we should remove the need for attribute if possible.